payton/block-templates/page.html

```diff
@@ -1,7 +1,7 @@
 <!-- wp:template-part {"slug":"header"} /-->
 
-<!-- wp:group {"style":{"spacing":{"padding":{"right":"20px","left":"20px"}}},"layout":{"inherit":true}} -->
```

Comment:
Why did you remove the side padding?

Reply:
It didn't seem to make a difference to the layout, so I removed it. Although I've just noticed that if the title goes onto multiple lines, this padding adds a bit more space. I'll add it back!